static/elements/chromedash-color-status.html

```diff
@@ -1,31 +1,45 @@
 <link rel="import" href="../bower_components/polymer/polymer.html">
 
-<polymer-element name="chromedash-color-status" attributes="value max"><!-- width height corner">-->
+<dom-module id="chromedash-color-status">
+  <link rel="import" type="css" href="../css/elements/chromedash-color-status.css">
   <template>
-    <link rel="stylesheet" href="../css/elements/chromedash-color-status.css">
-   <!--<style>
-      span {
-        width: {{width}}px;
-        height: {{height}}px;
-      }
-    </style>-->
     <span id="status"></span>
   </template>
-  <script>
-    Polymer({
-      max: 7,
-      width: 10,
-      height: 10,
-      updateColor: function() {
-        var h = Math.round(120 - this.value * 120 / this.max);
-        this.$.status.style.backgroundColor = 'hsl(' + h + ', 100%, 50%)';
+</dom-module>
+<script>
```

Comment:
I'll move them in. We'll need to keep the `is: name` until Vulcanize works with auto-id-inference.